aqua/emacs-mac-app/Portfile

```diff
@@ -182,7 +184,12 @@ variant metal description {Enable experimental Metal support} {
     configure.args-append  --with-mac-metal
 }
 
-default_variants-append +nativecomp +treesitter
+variant renameapp description {Rename app bundle to avoid conflict with emacs-app} {
+    conflicts-delete emacs-app
+    conflicts-delete emacs-app-devel
```

Comment:
Most *-append/delete commands allow multiple arguments; is that not the case here?

Reply:
That does seem to be the case; I was not aware. I will fix it in the commit that addresses your other comment.